packages/wizard/src/lib/Footer/styles.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from '@emotion/styled';
+import { Box } from '@mui/material';
+
+export const BoxContainer = styled(Box)`
```

Comment:
we can use here flex with justifyContent: space-between

Reply:
done!